server/core/config/raw/policies.go

```diff
@@ -2,15 +2,16 @@ package raw
 
 import (
```

Comment:
Please add unit tests to each modified file if possible

Reply:
👍